src/main/resources/templates/db_scripts/schema.json

```diff
@@ -366,7 +361,7 @@
     {
       "tableName": "fund_distribution",
       "fromModuleVersion": "mod-finance-storage-4.2.1",
-      "mode": "delete",
+      "mode": "DELETE",
```

Comment:
What do you think about delete declaration completely instead of using "mode"" : "DELETE" ? It should work in the same way but schema.json will be simpler

Reply:
Are you sure the tables would get deleted ?
If yes, I feel like this can still be useful for documentation purpose (for instance if we hear a report that some customer using an old version has some data in a particular table and that table is no longer used, we will figure it out faster when seeing that table with the DELETE mode in the schema), but we could probably remove deleted tables from 4 versions back or more.

Reply:
yes it should be deleted but I am ok alsot remain as is

Reply:
Now that I think about it, I see another reason to keep this code: it helps to make sure we don't create a new table with the same name as an old table, which could potentially cause issues when upgrading FOLIO.